_posts/2020-09-19-cycle-16-results.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Bisq DAO Cycle 16: Results"
+author: Steve Jain
+excerpt: "Cycle 16 of the Bisq DAO ended at block 646,626 on September 02 2020. This post covers its results. <br><br>"
+lang: en
+---
+
+This post summarizes the results of Cycle 16 of the Bisq DAO.
+
+### Summary
+
+* Cycle took place between blocks 641,947 and 646,626
+  * Calendar dates: 08/02/2020 - 09/02/2020
```

Comment:
Europeans struggle with this data format a lot: I propose to use Aug 02 2020 - Sep 02 2020 or any variation that makes clear what months and days are.

Reply:
Good point, thanks—changed!